### PR TITLE
[DEV-126] Removing Pydantic Table from the Schema

### PR DESCRIPTION
**Pull request by cash1998**
- [x] Removed the logic where pydantic details are stored in postgres
- [x] Added logic to fetch the Pydantic class node details from neo4j
- [x] Logic to fetch classes from github repos
- [x] Deleted pydantic table   
- [x] Tested on local & docker

### CODE REVIEW

**Review by dhirenmathur — COMMENTED** [review]
Are duplicate class names handled? 
Are multiple class imports handled?

**Review by dhirenmathur — CHANGES_REQUESTED** [review]
Add error handling in neo4j queries

**Comment by cash1998** [comment]
> Are duplicate class names handled? 
No, the older code also did not tackle this issue, pondered a lot and seemed this was the best solution as for now

> Are multiple class imports handled?
Yes, it is being handled.

File: server/utils/parse_helper.py [hunk 1]
```diff
@@ -105,7 +105,6 @@ def reparse_cleanup(project_details, user_id):
     directory = project_details[1]
     project_id = project_details[2]
     EndpointManager(directory).delete_endpoints(project_id, user_id)
-    EndpointManager(directory).delete_pydantic_entries(project_id, user_id)
```
Comment (dhirenmathur) [inline server/utils/parse_helper.py:108]: pleaase delete pydantic reference frommknowledge graph file as well

**Review by dhirenmathur — CHANGES_REQUESTED** [review]

**Review by dhirenmathur — DISMISSED** [review]

**Review by dhirenmathur — DISMISSED** [review]

**Review by dhirenmathur — APPROVED** [review]

**Pull request merged**